internal/netconf/netobjects.go

```diff
@@ -61,11 +62,12 @@ type (
 		TunnelIP string
 	}
 
-	// EVPNIface represents the information required to render EVPN interfaces configuration.
-	EVPNIface struct {
-		VRF   VRF
-		SVI   SVI
-		VXLAN VXLAN
+	// Tenant represents the information required to render EVPN interfaces configuration.
```

Comment:
Why is this struct called tenant ?

Reply:
If a strict exists to render EVPN networks I would call it like this. I would not rename it to "Tenant".

Reply:
renamed it back to the original name